### DISCUSSION TITLE
Is that period supposed to be there?

### DISCUSSION THREAD

**Discussion by NickHodges** [vercel/community#4217] [Help]
### Summary

![image](https://github.com/vercel/community/assets/1060358/6c86d90f-a841-43ac-9443-2ac84de5a44c)

Is there supposed to be a period on the end of that domain server name..?

### Steps to Reproduce

1. Try to add a domain to your project
2. Notice that little period on the end of the domain server name and wonder if it is supposed to be there.
3. Ask here if it is.

**Comment by amyegan** [comment]
Hi @NickHodges! Yes, it is supposed to be there. 🙂 

Some services have configured their DNS record UI so that the trailing period is not needed, but many do need it to signify a fully qualified domain. This StackExchange post explains a bit more: https://serverfault.com/a/18126

**Reply by NickHodges** [reply]
Okay!  Thanks very much.
